Pipecontrol

davegermiquet opened the pull request:
Small problem with prompts when piping. However Let me know what you think.
Added GREP
Fixed Tail
Fixed Head
Did some changes to cat...

davegermiquet committed:
Added pipe implementation
changed some functionality with tail/head/cat to work with pipe implementation
Got rid of ;'s of some places where it should not be :)

micheloosterhof commented:
Hi!
Could you rebase on master please?
Also avatar.py has an extra '1' on line 2?



On 30 May 2016 at 03:36, Dave Germiquet <notifications@github.com> wrote:

> Small problem with prompts when piping. However Let me know what you think.
> Added GREP
> Fixed Tail
> Fixed Head
> Did some changes to cat...
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/micheloosterhof/cowrie/pull/176
> Commit Summary
>
>    - Added pipe implementation
>    - Finished up adding pipe. Still a problem with the prompt..
>    - Removed unneeded import
>    - Minor Grep Fix
>
> File Changes
>
>    - *M* cowrie/commands/apt.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-0> (4)
>    - *M* cowrie/commands/fs.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-1>
>    (248)
>    - *A* cowrie/core/StdOutStdErrEmulationProtocol.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-2> (43)
>    - *M* cowrie/core/avatar.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-3> (2)
>    - *M* cowrie/core/fs.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-4> (55)
>    - *M* cowrie/core/honeypot.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-5>
>    (127)
>    - *M* cowrie/core/protocol.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-6> (27)
>    - *M* cowrie/insults/insults.py
>    <https://github.com/micheloosterhof/cowrie/pull/176/files#diff-7> (2)
>
> Patch Links:
>
>    - https://github.com/micheloosterhof/cowrie/pull/176.patch
>    - https://github.com/micheloosterhof/cowrie/pull/176.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/micheloosterhof/cowrie/pull/176>, or mute the thread
> <https://github.com/notifications/unsubscribe/ABA4g0fcnBeGbz0q1RNKRpAj0pDjuaCSks5qGiL1gaJpZM4IpaAb>
> .
>

davegermiquet commented:
Ok.. Rebased.

davegermiquet committed:
Removed unneeded ;'s.
There is a small bug that needs to be fixed with the prompts showing multiple ties

davegermiquet commented:
Ok Let me know what you think.

davegermiquet committed:
Finally FIXED PROMPT BUG!
My fault, incorrect indentation in if pp statement

davegermiquet commented:
Ok. Everything works now. Even the prompt bug :)

davegermiquet committed:
Fixed a problem when you EXIT it removes the protocol.username
So if we are exiting don't show the PROMPT!!

davegermiquet commented:
Ok. Sorry for the last minute commits...this should be ready to merge if you want.
